Boolean.md

```diff
@@ -0,0 +1,84 @@
+##Boolean Type
+Boolean is a fancy term meaning something that is either true or false.  While numbers and math form the backbone of any piece of software, everything (and I do mean everything) in computers, software and microchips, breaks down to ones and zeros, true and false.  While it isn't important to understand why that is and how exactly it works, it is important to understand how boolean expressions work so that your code will be able to do something interesting like this:
+
+````Swift
+func finishCourse() {
+    if gradeIsPassing() {
+    	print("Hooray!")
+   } else {
+      print("Doh!")
+   }
+}
+````
+
+The ````gradeIsPassing()```` function takes one course of action if a true value is returned and a different course of action if a false value is returned.
+Almost every program you write will need to take different courses of action based on boolean values, so it's important to have a really good grasp of what booleans and boolean expressions are as well as how to use them.
+
+Swift recognizes a value as boolean if it sees ````true```` or ````false````.  You can implicitly declar a boolean variable ````let a = false```` or explicitly declare a boolean variable ````let i:Bool = true````.
```

Comment:
I like showing this kind of example, I like the idea of structuring it in a code snippet like so:

```swift
var a: Bool = true
// Explicitly declaring a variable called 'a' of type Bool to equal true


var b = false
// Implicitily delaring a variable called 'b' to equal false where 'b' is a variable of type Bool
```